plugins/modules/apt_rpm.py

```diff
@@ -102,6 +125,20 @@ def update_package_db(module):
     if rc != 0:
         module.fail_json(msg="could not update package db: %s" % err)
 
+    return out
+
+
+def clean(module):
+    rc, out, err = module.run_command([APT_PATH, "clean"], check_rc=True)
+
+    return "Cleaned\n"
+
+
+def dist_upgrade(module):
+    rc, out, err = module.run_command([APT_PATH, "-y", "dist-upgrade"], check_rc=True)
```

Comment:
How does this detect whether changes happened?

Reply:
It's planned but not in this commit because complex analysis of content will be needed.

Reply:
Adding a feature together with others which doesn't have proper change detection sounds a bit premature. Why not remove upgrade from this PR then and add it in a new PR together with proper change detection for it?

Reply:
You are right, I will make changes detection.

Reply:
I add modufication checks for all operations.